portfolio/code_journal.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+	<meta charset="utf-8">
+	<title>Pushpa's Portfolio Page</title>
+	<link rel="stylesheet" href="Styles/code_journal.css">
+
+</head>
+<body>
+
+
+
+
+	<main class="container">
+
+		<div class="navigator">
+			<nav>
+				<ul class ="linkslist">
+					<li><a href="index.html">Home</a></li>
```

Comment:
Do you need the `<div class="navigator">` here? It seems redundant with the `<nav>`.